scripts/battle/GameState.cs

```diff
@@ -70,6 +70,7 @@ public class GameState {
 
     public void StartTurn() {
         DeselectDeadEnemy();
+        HideDeadEnemies();
```

Comment:
Move to the end of `EndTurn()`

Reply:
These are here for the case of enemies knocking themselves out on their turn. Headbutt, blood and sand and all that. 
I don't think enemies use EndTurn(), so they'd remain selected at beginning of user turn if we move these.

Reply:
`EndTurn()` is what makes enemies take their turns, so we just need to run these after that and before the `StartTurn()` call.

Reply:
True enough, was laser focused on the enemy turn and missed that. 
Made the move